runtime/tests/integration_tests.rs

```diff
@@ -30,20 +47,84 @@ fn run_rt_test(test_bin_name: Option<&str>) {
     })
     .unwrap();
 
+    let image = caliptra_builder::build_and_sign_image(
+        &FMC_WITH_UART,
+        &runtime_fwid,
+        ImageOptions::default(),
+    )
+    .unwrap();
+
+    model.step_until(|m| m.soc_ifc().cptra_flow_status().read().ready_for_fw());
+
+    model.upload_firmware(&image.to_bytes().unwrap()).unwrap();
```

Comment:
nitpick: you can get this part for free if you pass set BootParams::fw_image to Some(&image.to_bytes().unwrap()) above. See https://github.com/chipsalliance/caliptra-sw/blob/main/hw-model/src/lib.rs#L185

Reply:
Done.